tests/lib/snaps/test-snapd-network-status-provider/consumer

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+
+import dbus
+import os
+import sys
+
+def _get_obj():
+    return dbus.SystemBus().get_object("com.ubuntu.connectivity1.NetworkingStatus", "/com/ubuntu/connectivity1/NetworkingStatus")
+
+def get_version():
+    obj = _get_obj()
+    print(obj.GetVersion(dbus_interface="com.ubuntu.connectivity1.NetworkingStatus"))
```

Comment:
I'll leave the same remark as before. It is weird that a `get_foo` function *prints* foo rather than returning it.

Reply:
Thanks, I refactored the python scripts.